.github/workflows/main.yml

```diff
@@ -26,30 +26,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.7, 3.9]
+        python: [3.7, 3.8, 3.9]
```

Comment:
Don't think this is required please check on comment on vaishnavi's pr

Reply:
I added this because sometimes code blocks failed for particular software/library version. We should test each version that we provide support for.